src/SFA.DAS.AssessorService.Web/ViewModels/Shared/StandardVersion.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace SFA.DAS.AssessorService.Web.ViewModels.Shared
+{
+    public class StandardVersion
+    {
+        public string Title { get; set; }
+        public string StandardUId { get; set; }
+        public string Version { get; set; }
+
+        public static implicit operator StandardVersion(Api.Types.Models.Standards.StandardVersion version)
```

Comment:
We usually use mappers for this sort of thing - I don't think this type of conversion is used anyway else in the code